build_tools/benchmarks/reporting/common/utils.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2022 The IREE Authors
```

Comment:
We are in the process of moving all python modules under build_tools to https://github.com/iree-org/iree/tree/main/build_tools/python. Could you also move this part to that directory?

Idea is keeping all python modules under `build_tools/python` and the executable scripts in their original path (like `build_tools/benchmarks/reporting/parse_shark_benchmarks.py`). Ideally we should also make the executable scripts as small as possible (they are basically trampolines). More details here #10065

We use the magic like https://github.com/iree-org/iree/blob/main/build_tools/benchmarks/generate_cmake_benchmark_suites.py#L13 to resolve the search path issue in the executable scripts.

Reply:
To `build_tools/python/reporting/common/utils.py`

Reply:
Done